pkg/installer/install.go

```diff
@@ -37,25 +38,26 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/retry"
+	"sigs.k8s.io/yaml"
 
 	operatorv1 "github.com/openshift/api/operator/v1"
 	operatorclient "github.com/openshift/client-go/operator/clientset/versioned"
-
 	securityclient "github.com/openshift/client-go/security/clientset/versioned"
+	installertypes "github.com/openshift/installer/pkg/types"
 
 	"github.com/openshift-hive/hypershift-installer/pkg/api"
+	"github.com/openshift-hive/hypershift-installer/pkg/assets"
 	"github.com/openshift-hive/hypershift-installer/pkg/ignition"
 	"github.com/openshift-hive/hypershift-installer/pkg/pki"
 	"github.com/openshift-hive/hypershift-installer/pkg/render"
-
-	"github.com/openshift-hive/hypershift-installer/pkg/assets"
 )
 
 const (
 	externalOauthPort     = 8443
 	workerMachineSetCount = 3
 
-	defaultControlPlaneOperatorImage = "quay.io/hypershift/hypershift-operator:latest"
+	defaultControlPlaneOperatorImage = "registry.svc.ci.openshift.org/hypershift-toolkit/ibm-roks-4.4:control-plane-operator"
```

Comment:
default will be a CI build?

Reply:
Yes, currently control-plane-operator images are built by CI as soon as a PR merges. So that is the "canonical" repo for now, since they are not pushed externally anywhere.